commands/cfe.js

```diff
@@ -1,8 +1,49 @@
+/* eslint-disable max-len */
+/* eslint-disable prettier/prettier */
 'use strict';
 
+// eslint-disable-next-line object-curly-spacing
+const { MessageEmbed } = require('discord.js');
 const Validation = require('../utils/validator');
 const Scrapper = require('../webScraping/cfeScrapper');
 const myValidator = new Validation();
+const fs = require('fs');
+
+// eslint-disable-next-line valid-jsdoc
+/**
+ * This function create the message that will send the bot
+ * @param { object } cfeScrapper Instance of Scrapper
+ * @param { string } route Where it will save the file created
+ * @param {*} message
+ */
+async function sendTableMessage(cfeScrapper, route, message) {
+  const scrap = await cfeScrapper.doScraping(route);
+  if (scrap === false) {
+    message.channel.send('There is no data available');
+    return;
+  }
+  fs.readFile(route, (err, data) => {
+    if (err) throw err;
+    const dataFromJson = JSON.parse(data);
+    const embed = new MessageEmbed()
```

Comment:
why use Embed message ?

Reply:
Because I've not found another way to do it, I tried to use MD but I noticed that it only works with bold, italic... but not for tables